### PR TITLE
People Update OH

### PR DESCRIPTION
**Pull request by bfossen-ce**
People Update OH

### CODE REVIEW

File: data/oh/retired/Beth-Liston-2d01ff0c-25a8-49a2-b571-ab3a7866fcf2.yml [hunk 1]
```diff
@@ -11,14 +11,13 @@ party:
 - name: Democratic
 roles:
 - start_date: '2019-01-01'
+  end_date: '2023-01-01'
   type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:oh/government
```
Comment (maxknutsen) [inline data/oh/retired/Beth-Liston-2d01ff0c-25a8-49a2-b571-ab3a7866fcf2.yml:16]: @bfossen-ce move to legislature.

File: data/oh/retired/Catherine-D-Ingram-94ba65dd-57c5-4ab0-b607-5e310d50cae0.yml [hunk 2]
```diff
@@ -10,14 +10,13 @@ image: https://www.legislature.ohio.gov/Assets/HouseMembers/Headshots/Medium/285
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2023-01-01'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:oh/government
   district: '32'
```
Comment (maxknutsen) [inline data/oh/retired/Catherine-D-Ingram-94ba65dd-57c5-4ab0-b607-5e310d50cae0.yml:16]: @bfossen-ce move to legislature.

File: data/oh/retired/Cecil-Thomas-5ba7ae7d-2f10-42b7-8784-8e734507ce33.yml [hunk 3]
```diff
@@ -7,17 +7,17 @@ image: http://www.ohiosenate.gov/Assets/Members/Headshots/Medium/2801.jpg
 party:
 - name: Democratic
 roles:
-- type: upper
+- end_date: '2023-01-01'
+  type: upper
   jurisdiction: ocd-jurisdiction/country:us/state:oh/government
   district: '9'
```
Comment (maxknutsen) [inline data/oh/retired/Cecil-Thomas-5ba7ae7d-2f10-42b7-8784-8e734507ce33.yml:13]: @bfossen-ce move to legislature.

File: data/oh/retired/Daniel-P-Troy-04d4dc4f-a38a-4d50-8962-ea961550b2ff.yml [hunk 4]
```diff
@@ -4,14 +4,13 @@ image: https://www.legislature.ohio.gov/Assets/HouseMembers/Headshots/Medium/294
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2023-01-01'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:oh/government
```
Comment (maxknutsen) [inline data/oh/retired/Daniel-P-Troy-04d4dc4f-a38a-4d50-8962-ea961550b2ff.yml:9]: @bfossen-ce move to leg

File: data/oh/retired/Darrell-Kick-92de07aa-a6c7-4c4b-ada7-02ebbe4a17f6.yml [hunk 5]
```diff
@@ -10,14 +10,14 @@ image: https://www.legislature.ohio.gov/Assets/HouseMembers/Headshots/Medium/285
 party:
 - name: Republican
 roles:
-- type: lower
+- end_date: '2023-01-01'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:oh/government
   district: '70'
```
Comment (maxknutsen) [inline data/oh/retired/Darrell-Kick-92de07aa-a6c7-4c4b-ada7-02ebbe4a17f6.yml:16]: @bfossen-ce move to leg

File: data/oh/retired/Dick-Stein-0e7997cd-9c4b-4f28-8ed5-1f9bda887e07.yml [hunk 6]
```diff
@@ -10,14 +10,13 @@ image: https://www.legislature.ohio.gov/Assets/HouseMembers/Headshots/Medium/285
 party:
 - name: Republican
 roles:
-- type: lower
+- end_date: '2023-01-01'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:oh/government
```
Comment (maxknutsen) [inline data/oh/retired/Dick-Stein-0e7997cd-9c4b-4f28-8ed5-1f9bda887e07.yml:15]: @bfossen-ce move to legislature.

File: data/oh/retired/Joseph-A-Miller-III-9b5a9ff9-b35a-49aa-aa64-5d2bab574843.yml [hunk 7]
```diff
@@ -13,14 +13,13 @@ party:
 - name: Democratic
 roles:
 - start_date: '2019-01-01'
+  end_date: '2023-01-01'
   type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:oh/government
```
Comment (maxknutsen) [inline data/oh/retired/Joseph-A-Miller-III-9b5a9ff9-b35a-49aa-aa64-5d2bab574843.yml:18]: @bfossen-ce move to leg.

File: data/oh/retired/Juanita-O-Brent-ede2c8b0-9c35-4ebf-9a4a-4349fe4cc224.yml [hunk 8]
```diff
@@ -10,14 +10,13 @@ party:
 - name: Democratic
 roles:
 - start_date: '2019-01-01'
+  end_date: '2023-01-01'
   type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:oh/government
   district: '12'
```
Comment (maxknutsen) [inline data/oh/retired/Juanita-O-Brent-ede2c8b0-9c35-4ebf-9a4a-4349fe4cc224.yml:16]: @bfossen-ce move to leg.

File: data/oh/retired/Kris-Jordan-6690fa34-8958-4578-a110-4d3efe3f3618.yml [hunk 9]
```diff
@@ -12,14 +12,13 @@ roles:
   jurisdiction: ocd-jurisdiction/country:us/state:oh/government
   district: '19'
 - start_date: '2019-01-01'
+  end_date: '2023-01-01'
   type: lower
```
Comment (maxknutsen) [inline data/oh/retired/Kris-Jordan-6690fa34-8958-4578-a110-4d3efe3f3618.yml:16]: @bfossen-ce move to leg.

File: data/oh/retired/Paula-Hicks-Hudson-3329274c-22b6-4b0a-b12a-e64c5733bef9.yml [hunk 10]
```diff
@@ -11,14 +11,14 @@ party:
 - name: Democratic
 roles:
 - start_date: '2019-01-01'
+  end_date: '2023-01-01'
   type: lower
```
Comment (maxknutsen) [inline data/oh/retired/Paula-Hicks-Hudson-3329274c-22b6-4b0a-b12a-e64c5733bef9.yml:15]: @bfossen-ce move to legislature.

File: data/oh/retired/Rodney-Creech-c34a852a-71fc-4b21-a594-bc4a75b799c8.yml [hunk 11]
```diff
@@ -6,14 +6,14 @@ image: https://www.legislature.ohio.gov/Assets/HouseMembers/Headshots/Medium/294
 party:
 - name: Republican
 roles:
-- type: lower
+- end_date: '2023-01-01'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:oh/government
   district: '43'
```
Comment (maxknutsen) [inline data/oh/retired/Rodney-Creech-c34a852a-71fc-4b21-a594-bc4a75b799c8.yml:12]: @bfossen-ce move to leg.

File: data/oh/legislature/Thomas-Hall-7948afad-ec3c-4e4c-a073-305fb8cd1b21.yml [hunk 12]
```diff
@@ -1,20 +0,0 @@
-id: ocd-person/7948afad-ec3c-4e4c-a073-305fb8cd1b21
-name: Thomas Hall
-given_name: Thomas
-family_name: Hall
-image: https://www.legislature.ohio.gov/Assets/HouseMembers/Headshots/Medium/2947.jpg
-party:
-- name: Republican
-roles:
-- type: lower
```
Comment (maxknutsen) [inline data/oh/legislature/Thomas-Hall-7948afad-ec3c-4e4c-a073-305fb8cd1b21.yml:9]: @bfossen-ce this file should not be deleted. This person ended their time in district 53 on 01-01-2023 but began serving in house district 46 on the same date.

File: data/oh/retired/Casey-Weinstein-d7949531-f267-4e99-9611-87b3ec3e04ac.yml [hunk 13]
```diff
@@ -10,22 +10,26 @@ biography: "In Casey\u2019s family, public service is a way of life. His grandfa
...
-offices:
-- classification: capitol
-  address: 77 South High Street; 10th Floor; Columbus, OH 43215
-  voice: 614-466-1177
-  fax: 614-719-6942
+- start_date: '2023-01-01'
+  type: lower
+  jurisdiction: ocd-jurisdiction/country:us/state:oh/government
+  district: '34'
 links:
```
Comment (maxknutsen) [inline data/oh/retired/Casey-Weinstein-d7949531-f267-4e99-9611-87b3ec3e04ac.yml:26]: @bfossen-ce move to leg.

File: data/oh/retired/Kent-Smith-d9c3dfce-8b7f-402f-9dec-5179e38b528f.yml [hunk 14]
```diff
@@ -3,18 +3,18 @@ name: Kent Smith
...
-image: https://www.legislature.ohio.gov/Assets/HouseMembers/Headshots/Medium/2804.jpg
+image: https://www.legislature.ohio.gov/assets/people/kent-smith/headshots/thumbnails/medium/kent-smith-january-2021-headshot_medium.jpg
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2023-01-01'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:oh/government
   district: '8'
```
Comment (maxknutsen) [inline data/oh/retired/Kent-Smith-d9c3dfce-8b7f-402f-9dec-5179e38b528f.yml:13]: @bfossen-ce move to leg.

**Review by maxknutsen — COMMENTED** [review]
@bfossen-ce some folks need moving from leg to retired or vice versa and one deleted entry that needs to not be deleted. Other than that all set!

**Comment by bfossen-ce** [comment]
> @bfossen-ce some folks need moving from leg to retired or vice versa and one deleted entry that needs to not be deleted. Other than that all set!

I have updated these requests and we should be good to go

**Review by maxknutsen — APPROVED** [review]

**Pull request merged**